src/api/config/config.go

```diff
@@ -21,9 +21,10 @@ type configurationWithOverrides map[string]Configuration
 type Configuration struct {
 	Cors                 cors.Cors
 	Repos                repos.Repos
-	ReleasesEnabled      bool  `yaml:"releasesEnabled"`
-	TillerPortForward    bool  `yaml:"tillerPortForward"`
-	CacheRefreshInterval int64 `yaml:"cacheRefreshInterval"`
+	ReleasesEnabled      bool   `yaml:"releasesEnabled"`
+	TillerNameSpace      string "kube-system"
```

Comment:
We should rename this to TillerNamespace.

Go doesn't let you set defaults for a struct field (unfortunately), so the only way to do this would be to change `GetConfig()` to set TillerNamespace = "kube-system" if it's not set. The unset value for a string is simply empty string for Go, so a simple if statement such as works:

```
if conf.TillerNamespace == "" {
  conf.TillerNamespace = "kube-system"
}
```

This should be set in GetConfig() after reading in the config from the file.

You should update the field definition to:
```
TillerNamespace      string `yaml:"tillerNamespace"`
```